sql/driver/driver_it_test.go

```diff
@@ -442,6 +442,53 @@ func TestSQLQueryWithQueryTimeout(t *testing.T) {
 	testSQLQuery(t, ctx, "int", "int", fn, fn)
 }
 
+func TestSetLoggerConfig(t *testing.T) {
+	if err := driver.SetLoggerConfig(&logger.Config{Level: logger.ErrorLevel}); err != nil {
+		t.Fatal(err)
+	}
+	if err := driver.SetLoggerConfig(nil); err != nil {
+		t.Fatal(err)
+	}
+}
+
+func TestSetSSLConfig(t *testing.T) {
+	if err := driver.SetSSLConfig(&cluster.SSLConfig{Enabled: true}); err != nil {
+		t.Fatal(err)
+	}
+	if err := driver.SetSSLConfig(nil); err != nil {
+		t.Fatal(err)
+	}
+}
+
+func TestContextCancelAfterFirstPage(t *testing.T) {
+	/*
+		This test ensures context cancellation works for results after the first page (fetch operation).
+		generate_stream function generates N rows per second. Since the context times out after 1100 ms, it should be canceled right after getting the first number.
+	*/
+	it.SkipIf(t, "hz < 5.0")
+	it.SQLTester(t, func(t *testing.T, client *hz.Client, config *hz.Config, m *hz.Map, mapName string) {
+		db := driver.Open(*config)
+		defer db.Close()
+		ctx, cancel := context.WithCancel(context.Background())
+		defer cancel()
+		tic := time.Now()
```

Comment:
we can now get rid of the timing stuff right?

Reply:
No, we still want to be sure that this test takes about 1 sec.